Remotion/ObjectBinding/Web/UI/Controls/BocListImplementation/Rendering/BocColumnRendererBase.cs

```diff
@@ -45,6 +47,12 @@ public abstract class BocColumnRendererBase<TBocColumnDefinition> : IBocColumnRe
     /// <summary>Filename of the image used to indicate an descending sort order of the column in its title cell.</summary>
     protected const string c_sortDescendingIcon = "sprite.svg#SortDescending";
 
+    /// <summary>Filename of the image used to indicate that there are validation failures in a cell.</summary>
+    protected string c_validationIndicatorIcon = "sprite.svg#ValidationError";
+
+    /// <summary>Filename of the image used to indicate that there are validation failures in a cell.</summary>
+    protected string c_validationIndicatorPlaceholderIcon = "sprite.svg#ValidationErrorPlaceholder";
```

Comment:
We have a simpler option. if we actually do need to render an image instead of just a span with a size, we always use the "Spacer.gif" for this. See IconInfo.CreateSpacer